src/Kunstmaan/ConfigBundle/Helper/Menu/ConfigMenuAdaptor.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Kunstmaan\ConfigBundle\Helper\Menu;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Kunstmaan\AdminBundle\Helper\Menu\MenuItem;
+use Kunstmaan\AdminBundle\Helper\Menu\MenuAdaptorInterface;
+use Kunstmaan\AdminBundle\Helper\Menu\MenuBuilder;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+class ConfigMenuAdaptor implements MenuAdaptorInterface
+{
+    /**
+     * @var ContainerInterface $container
+     */
+    private $container;
+
+    /**
+     * @param ContainerInterface $container
+     */
+    public function __construct($container)
```

Comment:
pass kunstmaan_config as parameter and security.authorization_checker as services instead of container